### ISSUE TITLE
Split

### ISSUE THREAD

**Issue by rshkv** [AlexImmer/run-dmc#128]
You get a train and a test `DataFrame`.
Split the test set and for each split create a corresponding training DataFrame with dropped columns.

Please document your splitting criteria (what is known/unknown in each split) so the classifier people know what they're dealing with.

Return format:

``` python
[
    ('split_name', train_df, test_df),
    ('split_name2', train_df2, test_df2,
    ...
]
```
